src/effects/effectbuttonparameterslot.cpp

```diff
@@ -107,3 +109,33 @@ void EffectButtonParameterSlot::slotValueChanged(double v) {
         m_pEffectParameter->setValue(v);
     }
 }
+
+QDomElement EffectButtonParameterSlot::toXML(QDomDocument* doc) const {
+    QDomElement parameterElement;
+    if (m_pEffectParameter != nullptr) {
+        parameterElement = doc->createElement(EffectXml::Parameter);
+        XmlParse::addElement(*doc, parameterElement,
+                             EffectXml::ParameterId,
+                             m_pEffectParameter->id());
+        // TODO(rryan): Do smarter QVariant formatting?
```

Comment:
This TODO seams to be obsolete. addElement() requires QString and QString::number suits for that.